packages/react/src/transactions/adapters/PublicationCacheManager.ts

```diff
@@ -25,4 +68,19 @@ export class PublicationCacheManager {
       },
     );
   }
+
+  private async request(request: RequestPublicationArgs, fetchPolicy: FetchPolicy) {
+    const { data } = await this.client.query<GetPublicationData, GetPublicationVariables>({
+      query: GetPublicationDocument,
+      variables: {
+        request,
+        observerId: null, //session?.type === SessionType.WithProfile ? session.profile.id : null,
```

Comment:
shall we use `session.type` here as in the commented out code?

Reply:
good catch... forgot to re-instate this.

Reply:
I am not going to add extensive test coverage around this... the whole need of passing current profile id to queries like this one will go away with Lens API v2.